lib/python/bind_slice_methods.h

```diff
@@ -197,6 +198,23 @@ void expect_implicit_dimension(const Sizes &dims) {
     throw except::DimensionError(msg);
   }
 }
+
+template <class T>
+T slice_by_list(const T &obj,
+                const std::tuple<Dim, std::vector<scipp::index>> &index) {
```

Comment:
Any reason for combining dim and index into a tuple?

Reply:
Yes, this is how `__getitem__` works. You never get two or more separate args.